webapi/Extensions/ServiceExtensions.cs

```diff
@@ -119,7 +170,7 @@ internal static IServiceCollection AddCorsPolicy(this IServiceCollection service
                     policy =>
                     {
                         policy.WithOrigins(allowedOrigins)
-                            .WithMethods("GET", "POST", "DELETE")
+                            .WithMethods("POST", "GET", "PUT", "DELETE")
```

Comment:
Can you just use AllowAnyMethod() here?

Reply:
This will include changes that are not needed by this PR.